src/devices/radiohead_ask.c

```diff
@@ -0,0 +1,178 @@
+#include "rtl_433.h"
+#include "pulse_demod.h"
+#include "util.h"
+
+// Note: some code here provide from RadioHead source code.
+// http://www.airspayce.com/mikem/arduino/RadioHead/index.html
```

Comment:
When the crc is fixed only the 6to4 code is from the RadioHead code. So that can then this note can be changed to that.